Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -89,13 +90,19 @@ else:
                 for x in expand('/{{chromosome}}/{{chromosome}}_{suffixTwo}.vcf.gz',
                     suffixTwo = combinedFiles)] if len(combinedFiles) != 0 else []
         output:
-            temp(f'{vcfdir}/{{chromosome}}_final.vcf.gz')
+            vcf = (f'{vcfdir}/{{chromosome}}_final.vcf.gz'),
+            idx = (f'{vcfdir}/{{chromosome}}_final.vcf.gz.csi')
         conda: "bcftools"
-        threads: 1
-        resources: cpus=1, mem_mb=4000, time_min=5
-        log: 'logs/rename_{chromosome}.log'
+        log: 'logs/Rename_{chromosome}.log'
+        resources: cpus=1, mem_mb=32000, time_min=30
         shell:
             """
-            bcftools view {input} -O z -o {output}
-            bcftools index {output}
+            if [ -h {input} ]; then
```

Comment:
This now checks if it's a symlink - if yes, it creates the symlink to the original file